src/languages/ru-RU.js

```diff
@@ -346,7 +346,6 @@ export default {
     light: "СВЕТЛЫЙ",
     dark: "ТЕМНЫЙ",
     "New Status Page": "Новая страница статуса",
-    "Show update if available": "Показывать доступные обновления",
```

Comment:
Hi, why deleted?

Reply:
Accidentally deleted the wrong duplicate